src/main/java/WeekdayName.java

```diff
@@ -0,0 +1,20 @@
+public class WeekdayName {
+
+    public String getWeekdayName (Weekday weekday) {
+
+        switch (weekday) {
+            case Monday : return "Monday";
+            case Tuesday : return "Tuesday";
+            case Wednesday : return "Wednesday";
+            case Thursday : return "Thursday";
+            case Friday : return "Friday";
+            case Saturday : return "Saturday";
+            case Sunday : return "Sunday";
```

Comment:
da się to zrobić jedną linijką. sprawdź jakie metody masz dostępne na obiekcie 'weekday'. :)

```
    public String getWeekdayName (Weekday weekday) {

     return weekday.[..];

}
```

Reply:
Widzę, że są co najmniej dwie możliwości : toString() albo name()